homeassistant/components/opensky/sensor.py

```diff
@@ -79,14 +77,17 @@ def setup_platform(
     """Set up the Open Sky platform."""
     latitude = config.get(CONF_LATITUDE, hass.config.latitude)
     longitude = config.get(CONF_LONGITUDE, hass.config.longitude)
+    radius = config.get(CONF_RADIUS, 0)
+    bounding_box = OpenSky.get_bounding_box(latitude, longitude, radius)
+    session = async_get_clientsession(hass)
+    opensky = OpenSky(session=session)
     add_entities(
         [
             OpenSkySensor(
                 hass,
                 config.get(CONF_NAME, DOMAIN),
-                latitude,
-                longitude,
-                config.get(CONF_RADIUS),
+                opensky,
+                bounding_box,
                 config.get(CONF_ALTITUDE),
```

Comment:
Based on the schema, the default 0 is always injected so it is always present.
```suggestion
                config[CONF_ALTITUDE],
```